integration-tests/resteasy-jackson/pom.xml

```diff
@@ -35,6 +35,10 @@
             <artifactId>rest-assured</artifactId>
             <scope>test</scope>
         </dependency>
+		<dependency>
+			<groupId>org.apache.commons</groupId>
+			<artifactId>commons-lang3</artifactId>
+		</dependency>
```

Comment:
Small nitpick: the indentation is off.

Reply:
Actually, the new test is in the wrong module. It should probably be added to the `quarkus-integration-test-rest-client-reactive` module instead of the `quarkus-integration-test-resteasy-jackson`.

Ideally however, this would be a test in https://github.com/quarkusio/quarkus/tree/main/independent-projects/resteasy-reactive/server/vertx/src/test/java/org/jboss/resteasy/reactive/server/vertx/test in the manner of the other tests that exist there.

Reply:
I just realized the same and moved everything to a new module resteasy-reactive-jackson. I'll have a look whether it fits better in the other package you sent.

Reply:
The package I sent is where tests like these are added for RESTEasy Reactive. There are a lot of examples in there which should give you a good sense of what needs to be done, but let me know if you need help.

Reply:
BTW: even with existing tests I always get the following error:

    java.lang.RuntimeException: java.util.ServiceConfigurationError: io.smallrye.config.SmallRyeConfigFactory: io.quarkus.runtime.configuration.QuarkusConfigFactory not a subtype

So I cannot really tell if it works. Any idea what could be the reason.

Reply:
Are you sure you have built all the modules and that you are running the tests in the correct module?

Reply:
A complete rebuild with the current main did the trick, ty :+1: